root/static/scripts/release-editor/trackParser.js

```diff
@@ -323,6 +329,13 @@ releaseEditor.trackParser = {
             return data;
         }
 
+        //Use custom delimeter as separator.
+        if (options.useCustomDelimiter && options.customDelimiter != "") {
+            // Escape most regex characters.
+          options.customDelimiter = options.customDelimiter.replace(/[.*?+^$(){}|[\]]/g, '\\$&');
```

Comment:
Can you use root/static/scripts/common/utility/escapeRegExp.js here?

Reply:
I guess not, since you don't want to escape backslashes. Did you consider allowing full RegExp syntax? Maybe there's a good reason not to, but a non-programmer probably won't know what `\t` is in either case.